Mu2eKinKal/inc/KKFit.hh

```diff
@@ -356,7 +356,7 @@ namespace mu2e {
                     double du = (pca.sensorPoca().Vect()-smid).R();
                     double doca = fabs(pca.doca());
                     double dsig = std::max(0.0,doca-strawradius_)/sqrt(pca.docaVar());
-                    if(doca < maxStrawDoca_ && dsig < maxStrawDocaCon_ && du < straw.halfLength()){
+                    if(doca < maxStrawDoca_ && dsig < maxStrawDocaCon_ && du < straw.halfLength() * maxStrawUpos_){
```

Comment:
So the test is relative to the straw length? Is that better than adding a fixed buffer?
What are you using for maxStrawUpos_ in VST reco?